docs/api-swagger.yaml

```diff
@@ -1188,78 +1083,38 @@ components:
         code: 0
         message: message
         status: status
-    AuthTokens:
-      type: object
-      properties:
-        accessToken:
-          type: string
-          description: JWT access token
-        refreshToken:
-          type: string
-          description: Refresh token
-        expiresIn:
-          type: integer
-          description: Access token expiration time in seconds
-          format: int32
-      description: Authentication tokens
-      example:
-        expiresIn: 0
-        accessToken: accessToken
-        refreshToken: refreshToken
     UserStatus:
       type: object
       properties:
-        user:
-          $ref: '#/components/schemas/User'
         permissions:
           type: array
-          description: List of user permissions
+          description: List of user permissions for resources and namespaces.<br>https://samuelus.atlassian.net/wiki/spaces/ZPI/pages/28147713/Dokumentacja+Struktury+JSON+dla+Uprawnie
           items:
-            type: string
+            $ref: '#/components/schemas/UserStatus_permissions'
+        user:
+          $ref: '#/components/schemas/UserStatus_user'
       description: User session details and privileges
       example:
         permissions:
-        - permissions
-        - permissions
+        - resource:
```

Comment:
I think we want to transfer it as more of a json structure like map[string]map[string][]string example - applies to the autogenerated api changes too:
{
  "namespace1": {
    "resource1": [
      "Create",
      "Read",
      "Update"
    ],
    "resource2": [
      "Read",
      "Delete"
    ]
  },
  "namespace2": {
    "resource1": [
      "Create",
      "List"
    ],
    "resource3": [
      "Read",
      "Update",
      "Delete"
    ]
  }
}

Reply:
This are changes from API v0.0.4, but yes, it needs to change. I will update it